rpm: add alert about dropped support (#508)
fluent/fluentd-docs-gitbook: installation/install-by-rpm.md

> We use `$releasever` for repository path in the script and `$releasever` should be the major version only like `"7"`. If your environment uses some other format like `"7.2"`, change it to the major version only or set up .repo file manually.

Reviewer: Update the example version.

```suggestion
We use `$releasever` for repository path in the script and `$releasever` should be the major version only like `"9"`. If your environment uses some other format like `"9.2"`, change it to the major version only or set up .repo file manually.
```

Author: Thanks, it should be.

{% hint style='danger' %}

> Since v5.0.4, RHEL 7 / CentOS 7 was not supported anymore because CentOS 7 had reached EOL (June, 2024). Please consider migrating to the other release.

Reviewer: I'm not sure, but it looks more natural to me to focus on the current status (tense).

```suggestion
Since v5.0.4, RHEL 7 / CentOS 7 is not supported anymore because CentOS 7 has reached EOL (June, 2024). Please consider migrating to another release.
```
